<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbAgregar_alu">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color:rgb(205, 187, 227);
	border-radius:10px;
	border:10px;
}



QPushButton:hover{
background-color: rgb(170, 255, 127);
	border-radius:10px;
	border: 10px;
}
</string>
    </property>
    <property name="text">
     <string>AGREGAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../imagenes/AGREGARM.png</normaloff>../imagenes/AGREGARM.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pbConsultar_alu">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color:rgb(205, 187, 227);
	border-radius:10px;
	border:10px;
}


QPushButton:hover{
background-color: rgb(170, 255, 127);
	border-radius:10px;
	border: 10px;
}
</string>
    </property>
    <property name="text">
     <string>CONSULTAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../imagenes/CONSULTAR.png</normaloff>../imagenes/CONSULTAR.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pbModificar_alu">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color:rgb(205, 187, 227);
	border-radius:10px;
	border:10px;
}



QPushButton:hover{
background-color: rgb(170, 255, 127);
	border-radius:10px;
	border: 10px;
}
</string>
    </property>
    <property name="text">
     <string>ACTUALIZAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../imagenes/ACTUALIZAR.png</normaloff>../imagenes/ACTUALIZAR.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEliminar_alu">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color:rgb(205, 187, 227);
	border-radius:10px;
	border:10px;
}


QPushButton:hover{
background-color: rgb(170, 255, 127);
	border-radius:10px;
	border: 10px;
}
</string>
    </property>
    <property name="text">
     <string>ELIMINAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../imagenes/ELIMINAR.png</normaloff>../imagenes/ELIMINAR.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>34</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe MDL2 Assets</family>
      <pointsize>38</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Alumnos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMenu">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color:rgb(205, 187, 227);
	border-radius:10px;
	border:10px;
}


QPushButton:hover{
background-color: rgb(170, 255, 127);
	border-radius:10px;
	border: 10px;
}
</string>
    </property>
    <property name="text">
     <string>MENÃš</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../imagenes/icono_home.png</normaloff>../imagenes/icono_home.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>491</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagenes/Logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>251</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagenes/alumnos.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>pbAgregar_alu</zorder>
   <zorder>pbModificar_alu</zorder>
   <zorder>pbConsultar_alu</zorder>
   <zorder>pbEliminar_alu</zorder>
   <zorder>label</zorder>
   <zorder>pbMenu</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pbAgregar_alu</tabstop>
  <tabstop>pbEliminar_alu</tabstop>
  <tabstop>pbConsultar_alu</tabstop>
  <tabstop>pbModificar_alu</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
